packages/bacula-fd/build.sh

```diff
@@ -3,10 +3,11 @@ TERMUX_PKG_DESCRIPTION="Bacula backup software"
 TERMUX_PKG_LICENSE="AGPL-V3"
 TERMUX_PKG_MAINTAINER="Matlink <matlink@matlink.fr>"
 TERMUX_PKG_VERSION=11.0.5
-TERMUX_PKG_REVISION=3
+TERMUX_PKG_REVISION=4
 TERMUX_PKG_SRCURL=https://sourceforge.net/projects/bacula/files/bacula/${TERMUX_PKG_VERSION}/bacula-${TERMUX_PKG_VERSION}.tar.gz
 TERMUX_PKG_SHA256=ef5b3b67810442201b80dc1d47ccef77b5ed378fe1285406f3a73401b6e8111a
-TERMUX_PKG_DEPENDS="openssl"
+TERMUX_PKG_DEPENDS="openssl, liblzo"
+TERMUX_PKG_BUILD_DEPENDS="liblzo"
```

Comment:
No need to add liblzo to both BUILD_DEPENDS and DEPENDS, DEPENDS should be enough and correct

Reply:
Should be good now